public/javascripts/script.js

```diff
@@ -1,4889 +0,0 @@
-  "wowmail.com",
-  "wowway.com",
-  "wp.pl",
-  "wptamail.com",
-  "wrexham.net",
-  "writeme.com",
-  "writemeback.com",
-  "wrongmail.com",
-  "wtvhmail.com",
-  "wwdg.com",
-  "www.com",
-  "www.e4ward.com",
-  "www2000.net",
-  "wx88.net",
-  "wxs.net",
-  "x-mail.net",
-  "x-networks.net",
-  "x5g.com",
-  "xagloo.com",
-  "xaker.ru",
-  "xing886.uu.gl",
-  "xmastime.com",
-  "xms.nl",
-  "xmsg.com",
-  "xoom.com",
-  "xoxox.cc",
-  "xpressmail.zzn.com",
-  "xs4all.nl",
-  "xsecurity.org",
-  "xsmail.com",
-  "xtra.co.nz",
-  "xuno.com",
-  "xww.ro",
-  "xy9ce.tk",
-  "xyzfree.net",
-  "xzapmail.com",
-  "y7mail.com",
-  "ya.ru",
-  "yada-yada.com",
-  "yaho.com",
-  "yahoo.ae",
-  "yahoo.at",
-  "yahoo.be",
-  "yahoo.ca",
-  "yahoo.ch",
-  "yahoo.cn",
-  "yahoo.co",
-  "yahoo.co.id",
-  "yahoo.co.il",
-  "yahoo.co.in",
-  "yahoo.co.jp",
-  "yahoo.co.kr",
-  "yahoo.co.nz",
-  "yahoo.co.th",
-  "yahoo.co.uk",
-  "yahoo.co.za",
-  "yahoo.com",
-  "yahoo.com.ar",
-  "yahoo.com.au",
-  "yahoo.com.br",
-  "yahoo.com.cn",
-  "yahoo.com.co",
-  "yahoo.com.hk",
-  "yahoo.com.is",
-  "yahoo.com.mx",
-  "yahoo.com.my",
-  "yahoo.com.ph",
-  "yahoo.com.ru",
-  "yahoo.com.sg",
-  "yahoo.com.tr",
-  "yahoo.com.tw",
-  "yahoo.com.vn",
-  "yahoo.cz",
-  "yahoo.de",
-  "yahoo.dk",
-  "yahoo.es",
-  "yahoo.fi",
-  "yahoo.fr",
-  "yahoo.gr",
-  "yahoo.hu",
-  "yahoo.ie",
-  "yahoo.in",
-  "yahoo.it",
-  "yahoo.jp",
-  "yahoo.nl",
-  "yahoo.no",
-  "yahoo.pl",
-  "yahoo.pt",
-  "yahoo.ro",
-  "yahoo.ru",
-  "yahoo.se",
-  "yahoofs.com",
-  "yalla.com",
-  "yalla.com.lb",
-  "yalook.com",
-  "yam.com",
-  "yandex.com",
-  "yandex.pl",
-  "yandex.ru",
-  "yandex.ua",
-  "yapost.com",
-  "yapped.net",
-  "yawmail.com",
-  "yeah.net",
-  "yebox.com",
-  "yehey.com",
-  "yemenmail.com",
-  "yepmail.net",
-  "yert.ye.vc",
-  "yesey.net",
-  "yifan.net",
-  "ymail.com",
-  "yogotemail.com",
-  "yomail.info",
-  "yopmail.com",
-  "yopmail.pp.ua",
-  "yopolis.com",
-  "yopweb.com",
-  "youareadork.com",
-  "youmailr.com",
-  "your-house.com",
-  "your-mail.com",
-  "yourinbox.com",
-  "yourlifesucks.cu.cc",
-  "yourlover.net",
-  "yourname.freeservers.com",
-  "yournightmare.com",
-  "yours.com",
-  "yourssincerely.com",
-  "yoursubdomain.zzn.com",
-  "yourteacher.net",
-  "yourwap.com",
-  "yuuhuu.net",
-  "yyhmail.com",
-  "z1p.biz",
-  "za.com",
-  "zahadum.com",
-  "zaktouni.fr",
-  "zeepost.nl",
-  "zetmail.com",
-  "zhaowei.net",
-  "zhouemail.510520.org",
-  "ziggo.nl",
-  "zionweb.org",
-  "zip.net",
-  "zipido.com",
-  "ziplip.com",
-  "zipmail.com",
-  "zipmail.com.br",
-  "zipmax.com",
-  "zmail.ru",
-  "zoemail.com",
-  "zoemail.org",
-  "zoho.com",
-  "zohomail.com",
-  "zomg.info",
-  "zonnet.nl",
-  "zoominternet.net",
-  "zubee.com",
-  "zuvio.com",
-  "zuzzurello.com",
-  "zwallet.com",
-  "zweb.in",
-  "zxcv.com",
-  "zxcvbnm.com",
-  "zybermail.com",
-  "zydecofan.com",
-  "zzn.com",
-  "zzom.co.uk",
-  "zzz.com",
-  "zzz.pl",
-];
-
```

Comment:
Not a great idea to nest some code that you're using under a massive array, you don't want to make it hard to find code you're looking for...